api/v1alpha1/webhooks/release/webhook_test.go

```diff
@@ -84,4 +101,16 @@ var _ = Describe("Release validation webhook", func() {
 			Expect(webhook.ValidateDelete(ctx, release)).To(BeNil())
 		})
 	})
+
+	When("Default method is called", func() {
+		It("should set GracePeriodDays to ReleasePlan's value and return nil", func() {
+			ctx := context.Background()
+
+			Expect(k8sClient.Create(ctx, releasePlan)).Should(Succeed())
+			Expect(k8sClient.Create(ctx, release)).Should(Succeed())
+
+			Expect(webhook.Default(ctx, release)).To(BeNil())
+			Expect(release.Spec.GracePeriodDays).To(Equal(7))
```

Comment:
Might be nice to replace 7 with releasePlan.Spec.ReleaseGracePeriodDays

Reply:
done.